docs/reference/modules/global.md

```diff
@@ -0,0 +1,69 @@
+---
+extends: '@apostrophecms/piece-type'
+---
+
+# `@apostrophecms/global`
+
+**Alias:** `apos.global`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module populates the `req.data.global` with content or settings that are utilized site-wide. It is configured through "implicit sub-classing" of the schema fields present in the parent module. New fields are added and grouped identically to any widget or pieces module. The data from these fields is available to templates as `data.global`.
```

Comment:
The `req.data.global` object

Reply:
`@apostrophecms/global` is a piece type that only ever has one piece.

Reply:
Changed for the first and third comments. For the second comment, I'm assuming you are referring to the note?